src/components/Honeypots.tsx

```diff
@@ -30,9 +32,29 @@ export default function Honeypots() {
 
   const guestVaults: Array<JSX.Element> = [];
 
-  const vaults = vaultsData.map((vault: IVault) => {
-    // TODO: temp to not show guest vaults
-    if (!vault.parentVault.liquidityPool && vault.parentVault.registered && !vault.isGuest) {
+  useEffect(() => {
+    console.log("vaultsData", vaultsData);
+
+    const vaults = vaultsData.map((vault) => {
+      const { honeyPotBalance, stakingTokenDecimals, tokenPrice, apy } = vault.parentVault
+      console.log({ honeyPotBalance, stakingTokenDecimals, tokenPrice, apy })
+      if (tokenPrice !== undefined && apy !== undefined) {
+        return { vault, honeyPotBalance, stakingTokenDecimals, tokenPrice, apy }
+      }
+      return null
+    }).filter(vaultWithData => vaultWithData)
+      .sort((a, b) => {
+        return (b ? Number(fromWei(b.honeyPotBalance, b.stakingTokenDecimals)) * b.tokenPrice : 0) - (a ? Number(fromWei(a.honeyPotBalance, a.stakingTokenDecimals)) * a.tokenPrice : 0)
+      }).map(vaultWithData => vaultWithData!.vault)
+    setSortedVaults(vaults)
```

Comment:
@roienatan i do not think it is possible because the value of a vault can only be calculated from the price of the token which is queried from coingecko